src/main/java/io/khasang/pm/controller/AppController.java

```diff
@@ -30,11 +30,16 @@ public String getProjectPage() {
         return "project";
     }
 
-    @RequestMapping("/")
+    @RequestMapping("/getCat")
```

Comment:
this is not your functional - shoudn't change localhost:8080/ - for cat - localhost:8080/getCat
localhost:8080/getRole

solid -